about/package-scope.md

```diff
@@ -18,8 +18,6 @@ Currently, the packages that pyOpenSci reviews also need to fall into the
 technical and applied scope of our organization. This scope may expand over time
 as the organization grows.
 
-
-
 ## Is Your Package in Scope For pyOpenSci Review?
```

Comment:
@eriknw @NickleDave @arianesasso @cmarmo @sneakers-the-rat @Batalex i'd LOVE your input on these changes. this page is really the only one you have to look at in this review. the rest of the changes are just link fixes and cleanup. 

Many thanks in advance for this!